### ISSUE TITLE
make heading responsive

### ISSUE THREAD

**Issue by summerkoushal** [summerkoushal/construction#1]
<img width="1440" alt="Screenshot 2023-07-03 at 9 28 37 PM" src="https://github.com/summerkoushal/construction/assets/90128270/31ae4ad7-47bc-492a-a528-b0659bf3ad4d">
<img width="988" alt="Screenshot 2023-07-03 at 9 29 14 PM" src="https://github.com/summerkoushal/construction/assets/90128270/35fce12e-914e-4ac1-ab68-82ebd38f49b0">
<img width="414" alt="Screenshot 2023-07-03 at 9 29 39 PM" src="https://github.com/summerkoushal/construction/assets/90128270/f0514831-8cff-49f5-9b47-4e61b8421e38">

these are the screenshots for different screen sizes.
make them responsive with proper font sizes.

**Comment by Phil94comp** [comment]
I submitted a pull request. Please let me know if you like the changes.

**Comment by summerkoushal** [comment]
Actually i want that heading, to always be at the centre, so that at any viewport it looks fine. btw thanks for working on it.
also just adjusting the heading font size/ margin left would made it proper for certain resolutions only.
by any means a text-align: center; would be a perfect fit for it.

**Comment by pulkitxm** [comment]
Can I get assigned? let me know how to start

**Comment by summerkoushal** [comment]
ya sure @Pulkitxm, assigning task to you. do let me know if you have any issues regarding this.

**Comment by summerkoushal** [comment]
it doesnt fit my requirement. i want to have heading always at the center. (for any width size)

**Comment by Phil94comp** [comment]
![tablet-version-30px](https://github.com/summerkoushal/construction/assets/89309169/9b7f42a2-0975-42ea-b132-5861d5326326)
![tablet-version-25px](https://github.com/summerkoushal/construction/assets/89309169/e9772a9a-ce32-4df1-b4e4-05edc5704868)
![tablet-version-20px](https://github.com/summerkoushal/construction/assets/89309169/de75dc7a-655b-439d-8198-864b2c8999fa)
I figured out why the text is not centered on the minimum pixel count. The font size is larger than the minimum pixel size and when I made the font-size smaller the text looks more centered. The first screenshot is 30px, the second screenshot is 25px, and the third screenshot is 20px. I can fix the font-sizes, I just need to know what is the smallest font-size you want me to use for desktop, tablet, and mobile versions of the app?
